extensionsGallery-insider.json

```diff
@@ -3183,6 +3183,63 @@
 					],
 					"statistics": [],
 					"flags": "preview"
+				},
+				{
+					"extensionId": "65",
+					"extensionName": "machine-learning",
+					"displayName": "Machine Learning",
+					"shortDescription": "Machine Learning for SQL Databases",
```

Comment:
SQL databases? should it be SQL Server?

Reply:
this was the final description @MsSQLGirl ?